### ISSUE TITLE
get_stress with solid elements not working

### ISSUE THREAD

**Issue by Jdahn** [qd-cae/qd-eng#6]
Hi there,

I try to read out the stress in solid elements using following comands.

> d3plot = D3plot("d3plot",read_states=["disp","stress"])
> Element = d3plot.get_elementByID("solid",1)
> Element.get_stress() 

I expect an array with the size of 6x timeframes. However, my array is empty after these commands.
I tried both the element type 1 and 2 (LS-Dyna settings)

Best regards,

Jdahn

**Comment by qd-codie** [comment]
Hey,
it seems you did everything correctly. I found one bug, which seems to have happened from refactoring a while ago. I accidentally disable reading of solid stresses if there are no shell stresses. Do you get a warning; "Warning: There are no shell-stresses in the file." ? If so then this was the issue, and consider it fixed in 0.5.1 (fixed and tested!). Otherwise plz tell me so, then I have to keep looking (if you have an example I would be much faster).

As a note: my tests currently cover shells only (since my team solely relies on them). I will extend them, once I have a respective solid model.

**Comment by Jdahn** [comment]
Hi,
I indeed get the message "Warning: There are no shell-stresses in the file."...
I will update and try again.
Thanks!

**Comment by qd-codie** [comment]
Very cool, you're welcome. Note that I will only support python3 from 0.5.0 onwards.

**Comment by Jdahn** [comment]
Okay, That makes sense! I installed ananconda2 meaning that I was working with python 2.7. 
Running the pip command did not result in an upgrade to version 0.5.1.
This morning I installed anaconda3 with python version 3.6 and now I could install version 0.5.1 with pip.
as a side note: is still get the warning message:
> Warning: There are no shell-stresses in the file.

However, now it's possible to read out the stresses!
Thanks!
